### DISCUSSION TITLE
Instance segmentation output misses objects further away

### DISCUSSION THREAD

**Discussion by AniketDhar** [IDEA-Research/Grounded-Segment-Anything#393] [General]
I am trying out instance segmentation with GSA to identify traffic poles from a scene. The examples I tried manage to detect traffic poles that are close but totally misses the ones in the back. Is there any reason for this? Can I fix this with some configurable parameters?

`python grounded_sam_demo.py --config GroundingDINO/groundingdino/config/GroundingDINO_SwinT_OGC.py --grounded_checkpoint groundingdino_swint_ogc.pth --sam_checkpoint sam_vit_h_4b8939.pth --input_image assets/german_road_2.jpg --output_dir "outputs" --box_threshold 0.25 --text_threshold 0.2 --text_prompt "traffic pole" --device "cpu"`

![Picture1](https://github.com/IDEA-Research/Grounded-Segment-Anything/assets/25749369/f1d9e209-f13a-4285-9859-f43d03f73501)
![Picture2](https://github.com/IDEA-Research/Grounded-Segment-Anything/assets/25749369/acbff1c1-db34-4902-bff3-7fad4612ad9a)

**Comment by Nidacoke** [comment]
No label slacked

**Reply by AniketDhar** [reply]
What does it mean?
